test/e2e/crud.go

```diff
@@ -92,7 +92,12 @@ var _ = Describe("Federated", func() {
 					tl.Fatalf("Failed to create unlabeled resource in cluster %q: %v", clusterName, err)
 				}
 				clusterClient := genericclient.NewForConfigOrDie(clusterConfig)
-				defer clusterClient.Delete(context.TODO(), unlabeledObj, unlabeledObj.GetNamespace(), unlabeledObj.GetName())
+				defer func() {
+					err := clusterClient.Delete(context.TODO(), unlabeledObj, unlabeledObj.GetNamespace(), unlabeledObj.GetName())
+					if err != nil {
+						tl.Fatalf("Unexpected error: %v", err)
```

Comment:
Will `Fatalf` allow other deferred calls to execute?

Reply:
Yes, I have tested it.